README.md

```diff
@@ -66,6 +66,20 @@ List of Composite Actions:
 
 8. Tag: Commit the changes available in the workspace and tag the code as per the latest commit id. You have also the possibility of pushing to a tag only, by setting `push_tag_only` to `true`.
 
+9. Notification: Aims to send messages to a Microsoft Teams channel and slack channel to help users keep track of all the defined steps in a workflow, along with additional workflow details.
```

Comment:
The reporting step also writes to the summary. 

When you say
> Aims to send messages to a Microsoft Teams channel and slack channel

it should be `Aims to send messages to a Microsoft Teams channel and/or a slack channel ...`

Reply:
Updated.
Also Added tag `1.0.0` for team-notify-action

Reply:
I don't see any reference to the Summary part when doing the report step. And call it `Reporting` instead of `Notification`

Reply:
Updated the Reporting section